app/src/main/kotlin/com/wire/android/ui/common/CopyButton.kt

```diff
@@ -27,13 +28,14 @@ import com.wire.android.ui.common.button.WireSecondaryIconButton
 @Composable
 fun CopyButton(
     onCopyClicked: () -> Unit,
-    state: WireButtonState = WireButtonState.Default,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    @StringRes contentDescription: Int? = null,
+    state: WireButtonState = WireButtonState.Default
 ) {
     WireSecondaryIconButton(
         onButtonClicked = onCopyClicked,
         iconResource = R.drawable.ic_copy,
-        contentDescription = R.string.content_description_copy,
+        contentDescription = contentDescription ?: R.string.content_description_copy,
```

Comment:
i would be in favour of removing this implicit behaviour.with a default value for  @StringRes content description

```kotlin
@Composable
fun CopyButton(
    onCopyClicked: () -> Unit,
    state: WireButtonState = WireButtonState.Default,
    modifier: Modifier = Modifier
    modifier: Modifier = Modifier,
    @StringRes contentDescription: Int = R.string.content_description_copy,
    state: WireButtonState = WireButtonState.Default
) {
    WireSecondaryIconButton(
        onButtonClicked = onCopyClicked,
        iconResource = R.drawable.ic_copy,
        contentDescription = contentDescription,

```